modules/virt-configuring-disk-sharing-lun-cli.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-disk-sharing-lun-cli_{context}"]
+= Configuring disk sharing by using LUN and the command line
```

Comment:
I don't believe so. Neither the SME nor the QE reviewer made that request. This content for the command line was also reviewed extensively in CNV-22900 and is included in this PR only because there is a new UX way to accomplish this and I needed make the command line/UX distinction.

Reply:
Got it. I guess I was just wondering because a lot of times our documentation specifies this, and it relates to another peer review comments I left wondering about including the command for a specific step.

Reply:
If it is irrelevant feel free to disregard :)